src/Pages/CardReader/CardReader.js

```diff
@@ -282,21 +295,27 @@ export default function CardReader() {
   }
 
   return (
-    <div className='overview-container bg-white dark:bg-gradient-to-r dark:from-gray-800 dark:to-gray-600 min-h-[100dvh]'>
-      <h1 className='flex items-center justify-center text-gray-700 dark:text-white text-4xl font-bold py-4'>SCE Card Reader Page</h1>
-      <pre className='flex items-center justify-center text-gray-700 dark:text-white text-md py-2'>This webpage manages RFID cards used to unlock the office door in the SCE room</pre>
+    <div className='overview-container mx-5 dark:from-gray-800 dark:to-gray-600 min-h-[100dvh]'>
+      <h1 className='text-center text-4xl font-extrabold leading-none tracking-tight text-gray-700 md:text-5xl lg:text-6xl dark:text-white py-8'>SCE Card Reader Page</h1>
+      <pre className='whitespace-normal text-center max-w-[90%] mx-auto text-gray-700 dark:text-white font-normal py-2'>This webpage manages RFID cards used to unlock the office door in the SCE room</pre>
       <div className='flex flex-row items-center justify-center text-gray-700 dark:text-white text-xl font-bold pt-4'>
         <button
-          className={`p-2 hover:bg-gray-400 rounded-xl ${tab === 'registry' ? 'underline underline-offset-4' : ''}`}
-          onClick={() => handleTabChange('registry')}
+          className={`p-2 hover:bg-gray-400 ${selected === 'Card Registery' ? 'text-blue-500' : 'text-gray-700 dark:text-white'} rounded-xl ${tab === 'registry' ? 'underline underline-offset-4' : ''}`}
+          onClick={() => {
+            handleTabChange('registry');
+            setSelected('Card Registery');
```

Comment:
```suggestion
            setSelected('Card Registry');
```
i know its nitpicking but we wanna make sure there is no inconsistency with spelling or there might be errors